lighthouse-core/gather/driver.js

```diff
@@ -104,6 +104,10 @@ class Driver {
     return this._connection.disconnect();
   }
 
+  wsEndpoint() {
```

Comment:
copy/paste jsdoc or hopefully tsc smart enough?

Reply:
add docs for which endpoint this is? And I can't imagine who would think otherwise, but probably worth noting that it's CRI-only